int32 timeStamp
float32 batteryVoltage
int32 gyroRate
int32 health
